labman/db/process.py

```diff
@@ -3584,6 +3548,6 @@ def generate_prep_information(self):
             df = df[order]
             sio = StringIO()
             df.to_csv(sio, sep='\t', index_label='sample_name')
-            data[study] = sio.getvalue()
+            data[curr_prep_sheet_id] = sio.getvalue()
```

Comment:
This is outside of the scope of modifications in this PR but DataFrame.to_csv returns a string if no filepath or file handle is passed as the first argument. This imples that the intermediate StringIO object is not needed if that feature is used instead.

Reply:
👍